### PR TITLE
Update to Rerun 0.14

### PR DESCRIPTION
**Pull request by Wumpf**
update to rerun 0.14.0
Tested on mac arm64 (it just works!) using 0.14.0rc4

### CODE REVIEW

**Comment by Wumpf** [comment]
still waiting for conda forge PR to finish to the pixi command works with 0.14

File: pixi.toml [hunk 1]
```diff
@@ -5,15 +5,23 @@ channels = ["robostack-staging", "conda-forge"]
...
+    "ws",
+], cwd = "noetic_ws" }
+example_data = { cmd = "curl -L -C - -O http://rpg.ifi.uzh.ch/datasets/uzh-fpv-newer-versions/v3/indoor_forward_3_snapdragon_with_gt.bag", depends_on = [
+    "ws",
+], cwd = "noetic_ws/src/rerun_bridge" }
+example = { cmd = "bash -c 'source ./devel/setup.bash && roslaunch rerun_bridge example.launch'", depends_on = [
+    "build",
+    "example_data",
+    "ws",
+], cwd = "noetic_ws" }
```
Comment (emilk) [inline pixi.toml:24]: This is really weird formatting

Comment (Wumpf) [reply pixi.toml:24]: that's my auto formatter

Comment (emilk) [reply pixi.toml:24]: god damn toml…

**Comment by roym899** [comment]
Superseded by #4

**Pull request closed without merging**